commands/publish/__tests__/get-unpublished-packages.test.js

```diff
@@ -27,8 +33,38 @@ getPackument.mockImplementation(async pkg => {
 
 test("getUnpublishedPackages", async () => {
   const cwd = await initFixture("licenses-names");
-  const project = new Project(cwd);
+  const packages = await getPackages(cwd);
+  const packageGraph = new PackageGraph(packages);
+
+  const opts = new Map();
+  const pkgs = await getUnpublishedPackages(packageGraph, opts);
 
-  const pkgs = await getUnpublishedPackages(project, {});
-  expect(pkgs.map(p => p.name)).toEqual(["package-1", "package-3", "package-4", "package-5"]);
+  expect(pkgs).toMatchInlineSnapshot(`
```

Comment:
These changes are just fallout from unrelated refactoring (prompted by removal of a dependency I thought was clear). In the end, I didn't make the effort to separate it.

Reply:
Actually, I think I _will_ move this commit out of this PR. It's very noisy.